tools/ucd2c.pl

```diff
@@ -242,6 +243,24 @@ sub break_property {
     enumerated_property("auxiliary/${fname}BreakProperty",
         $pname, { Other => 0 }, 1, 1);
 }
+sub grapheme_cluster_break {
+    my ($fname, $pname) = @_;
+    enumerated_property("auxiliary/${fname}BreakProperty",
+        $pname, {
+            # Should not be set to Other for this one
+            Other => 0,
+            # Make sure we specify the values of Prepend and Extend
+            # We need these consistent since they are used in MoarVM code
+            #Prepend => 1,
+            #Extend => 2,
+            #E_Base => 3,
+            #E_Base_GAZ => 4,
+            #ZWJ        => 5,
+            #E_Modifier => 6,
+            #Glue_After_Zwj => 7
```

Comment:
It would cause inconsistent results for the values of the enum. And may have totally broken it by changing the return values to some other number. (It would assign the wrong 'name' to whatever happened in that enum)